OpenRA.Game/Settings.cs

```diff
@@ -33,6 +33,13 @@ public enum MPGameFilters
 		Incompatible = 16
 	}
 
+	[Flags]
+	public enum TextNotificationPoolFilters
+	{
+		None = 0,
+		Feedback = 1
```

Comment:
Perhaps this pool should be named "Transients" after all :thinking: I'm considering taking a shot at implementing a transient messages widget (discussed in #16355). "Feedback" and "UI Feedback" feels quite limited and is not very clear.

Reply:
After mulling over this for some more and watching some CNC3 footage I decided to leave it at "Feedback". In a follow up PR I plan to attempt a transient messages widget that combines UI feedback and transcriptions, similar to this:
![image](https://user-images.githubusercontent.com/1355810/121682558-e89df980-cac4-11eb-91da-fd82a4fdc8b9.png)

Reply:
See #19458